packages/create-app/templates/default-app/app-config.yaml.hbs

```diff
@@ -30,6 +31,13 @@ backend:
       password:
         $secret:
           env: POSTGRES_PASSWORD
+      # https://node-postgres.com/features/ssl
+      ssl:
+        #ca: # if you have a CA file and want to verify it you can uncomment this section
+        #  $secret:
+        #    file: <file-path>/ca/server.crt
+        $secret:
+          env: POSTGRES_SSL_OPTIONS # see https://www.postgresql.org/docs/current/libpq-ssl.html Table 33.1. SSL Mode Descriptions
```

Comment:
@timja - I'm curious what you are typically setting the `POSTGRES_SSL_OPTIONS` value to (e.g. `true`)? The `ssl` config is typically an object of keys or `true`. It's definitely not clear based on the `postgres` docs how this translates back to the `knex` / `node-postgres` from a single environment variable. From my local testing, the `$secret: env: VAR` won't auto parse to a JSON object, and I've been unable to get just `true` to work along (without using `NODE_TLS_REJECT_UNAUTHORIZED=0` or use `rejectUnauthorized: false`).

Reply:
@andrewthauer 

POSTGRES_SSL_OPTIONS=require

I linked to the docs on that line

Reply:
@andrewthauer Sorry for a naive question, but do you need to pass that particular thing in the env? Can't it be hardcoded to the actual value you know will be used in prod?

If there is a need to coerce env vars into strings, (e.g. the string `"true"` to `true`), we can do that - but it's a bit more complex in this case due to the field being able to take on several different shapes.

Reply:
Regarding unauthorized, I assume that would happen if the postgres server presented a certificate that node is not configured to trust (such as if it's self signed). I'm in deep water here since while I am familiar with SSL mechanics as such, I know little about the node trust store or how you configure postgres ssl, but I bet there is a lot of articles on those.

Reply:
@timja - Thanks for clarifying. Sorry, I clearly wasn't drawing the connection between the `node-postgres` and `postgres` docs on how to set this. I haven't seen any example which sets `ssl: require` although it does make sense :)  I'm used to passing this via the connection string `?sslmode=require` or `?ssl=true`.

@freben - Your correct. Just me testing locally with self signed certs that's all.

Reply:
```suggestion
          env: POSTGRES_SSL_OPTIONS # see https://www.postgresql.org/docs/current/libpq-ssl.html Table 33.1. SSL Mode Descriptions (e.g. `require`, `true`)
```

Reply:
```suggestion
          env: POSTGRES_SSL_OPTIONS # see https://www.postgresql.org/docs/current/libpq-ssl.html Table 33.1. SSL Mode Descriptions (e.g. POSTGRES_SSL_OPTIONS=require)
```

Reply:
Wondering the same thing as freben, couldn't we just make it

```yaml
connection:
  # https://node-postgres.com/features/ssl
  ssl: false # see https://www.postgresql.org/docs/current/libpq-ssl.html Table 33.1. SSL Mode Descriptions (e.g. require)
  # if you have a CA file and want to verify, use this section instead
  #ssl:
  #  ca:
  #    $secret:
  #      file: <file-path>/ca/server.crt
```

?

Reply:
Well can’t see anything that says you can use sslmode settings here, the official node Postgres docs don’t mention the values you need to go find the Postgres docs. But sure your suggestion works